docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -690,7 +690,7 @@ Status Codes:
 
 ### Resize a container TTY
 
-`POST /containers/(id)/resize?h=<height>&w=<width>`
+`POST /containers/(id)/resize`
```

Comment:
wondering: are both query parameters *required*? If they are, perhaps that was the reason for including them here (not sure)